presto-product-tests/README.md

```diff
@@ -43,6 +43,10 @@ broken.
 `hadoop-master`, `mysql`, `postgres`, `cassandra`, `presto-master`.
 They should point to your external IP address (shown by `ifconfig` on your Mac (not inside docker)).
 
+* The default memory setting of 2GB might not be sufficient for some profiles like singlenode-ldap.
+We recommend at least 4GB for such profiles. You can increase docker memory by going to Docker
```

Comment:
When i tried to run single node tests with `run_on_docker` i needed to slide up to 7 GB for them to pass.
 They definitely weren't passing at 4 GB.. but this was 1-2 mo ago, so maybe something changed or it was just mine machine.